src/tabs/classes.scss

```diff
@@ -2,18 +2,30 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import './index';
 
-.iui-tabs-horizontal {
-  @include iui-tabs-horizontal;
+.iui-tabs-wrapper {
+  position: relative;
 }
 
-.iui-tabs-borderless {
-  @include iui-tabs-borderless;
-}
+.iui-tabs {
+  @include iui-tabs;
 
-.iui-tabs-pill {
-  @include iui-tabs-pill;
-}
+  &.iui-default {
+    @include iui-tabs-default;
+  }
+
+  &.iui-borderless {
+    @include iui-tabs-borderless;
+  }
+  
+  &.iui-pill {
+    @include iui-tabs-pill;
+  }
+
+  &.iui-animated {
+    @include iui-tab-animated;
+  }
 
-.iui-tabs-content-area {
-  @include iui-tabs-content-area;
+  &.iui-not-animated {
```

Comment:
Do we need this class? What if `iui-animated` would reset not animated values?

Reply:
But then there will be tons of extra box-shadow values being overridden, and specificity could also be a problem.

Also I think `not-animated` option should not really be used when `animated` has the preferred styles. But making `animated` default also doesn't make sense because it requires JS. So we end up with separate classes 🤷

Reply:
Idk, then I would say to remove `not-animated` because there are some components that require JS e.g. `Toast`, `ExpandableBlock`.

Reply:
It's only few lines and does not have tests. Could be useful for blazor or something, idk